guvnor-asset-mgmt/guvnor-asset-mgmt-client/src/test/java/org/guvnor/asset/management/client/editors/repository/wizard/CreateRepositoryWizardTest.java

```diff
@@ -293,6 +285,69 @@ public void testManagedRepositoryMultiCompletedTest() {
         WizardTestUtils.assertWizardComplete( true, createRepositoryWizard );
     }
 
+    @Test
+    public void testCompletionStatusSuccess() {
+        setupNameAndOrgUnitMocks(true, ORGANIZATIONAL_UNIT);
+
+        createRepositoryWizard.start();
+
+        infoPage.onNameChange();
+        infoPage.onOUChange();
+
+        verify(view, times(2)).setCompletionStatus(false);
```

Comment:
I'm not saying this isn't correct; but can you advise why both ```setCompletionStatus( false )``` and ```setPageCompletionState( 0, false )``` are called twice (presumably) before ```setCompletionStatus( true )``` and ```setPageCompletionState( 0, true )``` is called once? I'd have expected one call for each (false then true).